package/MDAnalysis/analysis/base.py

```diff
@@ -173,7 +178,7 @@ def run(self, start=None, stop=None, step=None, verbose=None):
         logger.info("Choosing frames to analyze")
         # if verbose unchanged, use class default
         verbose = getattr(self, '_verbose', False) if verbose is None else verbose
-
+        if verbose == True : self.verbose = verbose
```

Comment:
Based on line 142, shouldn't this be `self._verbose`? 
Also, I'm probably getting the logic wrong here, but wouldn't it be best to always set `self._verbose` here? What if you have a case where you pass `False` to `run()` but `True` to `__init__`, in this construct `self._verbose` would be set to True even if verbose is currently `False` right?

Reply:
In case of parameter overriding for the run class can it be directly:
 verbose = getattr(self, '_verbose', False) if verbose is None else verbose
 -> self._verbose = verbose after the line

Reply:
Please do correct me if i am going wrong in understanding the requirement. (Just re-read the comments regarding deprecation.)